amf-shapes/shared/src/main/scala/amf/shapes/internal/spec/jsonschema/ref/JsonSchemaInference.scala

```diff
@@ -28,31 +29,15 @@ trait JsonSchemaInference {
 
   private def tryParserVersion(node: YNode)(errorHandler: AMFErrorHandler): Option[String] = {
     node.value match {
-      case scalar: YScalar => Some(adaptInput(scalar.text))
+      case scalar: YScalar => Some(scalar.text)
```

Comment:
Don't remove methods

Reply:
it's not needed, the same method exists in parent